### DISCUSSION TITLE
"Whats new"

### DISCUSSION THREAD

**Discussion by ryanraposo** [alefragnani/vscode-bookmarks#372] [Q&A] [answered]
Hey, @alefragnani :)

I really want to do something like the "What's new in Bookmarks" page, its so cool and well done!

I know its not a normal ask, but if you ever have some time to spare to go over building it, I'd appreciate that so much. 

I have a huge new feature coming out in my extension soon, and I want to make sure users who haven't opened up the sidebar in a while notice when it drops! [Here's a link.](https://github.com/ryanraposo/codeui/issues/24)

Thanks so much. (btw Bookmarks is one of the most well done extensions by far. Its just _right._ Such incredible work.)

**Comment by alefragnani** [comment, marked as answer]
Hi @ryanraposo ,

First of all, thank you for your kind words. I’m glad to know you like the extension and the What’s New page.

About that, yes, it’s already “exposed” as a separate repo. Just head to https://github.com/alefragnani/vscode-whats-new, follow the docs and enjoy 😁 

Also, once you have a Side Bar, maybe you would be interested on another NPM package I’ve created (https://www.npmjs.com/package/vscode-ext-help-and-feedback-view). If you ever used the Remove Development extensions on VS Code, you may be familiar with it

BTW, congratulations for your extension. Really interesting idea 👏 

Hope this helps

**Reply by ryanraposo** [reply]
@alefragnani I am floored! I really have to say, that was the best result/answer imaginable haha!

Thank you! For the good work you do, and for the kind words yourself. Take care.
